### ISSUE TITLE
cant use persian tts in goldenDict

### ISSUE THREAD

**Issue by sami-soft** [espeak-ng/espeak-ng#1386]
I used the Persian sounds in eSpeak in the Golden program, but unfortunately, the Golden program cannot properly pronounce the Persian text.

"C:\Program Files\eSpeak NG\espeak-ng.exe" -v fa %GDWORD%

I used the Persian sounds in eSpeak in the Golden program, but unfortunately, the Golden program cannot properly pronounce the Persian text.

"C:\Program Files (x86)\eSpeak\command_line\espeak.exe" -v mb-ir1 %GDWORD%

 
There is no problem with English words and pronounces them correctly.

**Comment by sami-soft** [comment]
I also tested it in the command line, there is a problem

**Comment by jaacoppi** [comment]
I think this is a Windows terminal issue. Works fine on linux:
```
$ espeak-ng -v fa 'سلام' -x
sal'Am
$ espeak-ng -v fa -f 1.txt -x
sal'Am

```

**Comment by sami-soft** [comment]
> I think this is a Windows terminal issue. Works fine on linux:
> 
> ```
> $ espeak-ng -v fa 'سلام' -x
> sal'Am
> $ espeak-ng -v fa -f 1.txt -x
> sal'Am
> ```

Yes, this problem is in the Windows terminal, but it also has this problem in the GoldenDict program that I am testing.

**Comment by jaacoppi** [comment]
Can the ticket be closed? Seems like this is not related to espeak-NG.

**Comment by sami-soft** [comment]
> Can the ticket be closed? Seems like this is not related to espeak-NG.

This problem is related to your software. If it is not related to the software, then where is the problem related to? You said that it is related to the Windows terminal, but I tested it in other programs and they had problems. So the problem is with this program itself.

**Comment by jaacoppi** [comment]
> This problem is related to your software. If it is not related to the software, then where is the problem related to? You said that it is related to the Windows terminal, but I tested it in other programs and they had problems. So the problem is with this program itself.

Sorry, mixed this with another thread. Your terminal might be using wrong encoding. Espeak-NG default to UTF8. Try -b to select encoding:
UTF8, default: espeak-ng -v fa 'سلام' -b 1 -x
8bit ascii: espeak-ng -v fa 'سلام' -b 2 -x
16bi: espeak-ng -v fa 'سلام' -b 4 -x

**Comment by sami-soft** [comment]
> espeak-ng -v fa 'سلام' -b 1 -x

Unfortunately, the problem was not solved

1. espeak-ng -v fa 'سلام' -b 1 -x   > result = null
2.  espeak-ng -v fa 'سلام' -b 2 -x   > result = (en)0'A: s'A:(fa)
3. espeak-ng -v fa 'سلام' -b 4 -x   > result = (ko)ph'at-tShjat-(fa)
4. د
